### DISCUSSION TITLE
Is it possible to have a list view filtered for the item on whose screen it appears?

### DISCUSSION THREAD

**Discussion by FloMiLe** [Budibase/budibase#1196] [Help]
I would like to create an inventory app. It should contain a table 'Items' that lists items with several item-specific details. Instead of only having a numbers field for the number of pieces of a specific item, I would like to store how many pieces are in different locations, respectively.

In Tadabase I would create a table like 'Pieces in Locations' with three columns: a relationship column for the item, a numeric column for the number of pieces, and a relationship column for the location. On the screen for an item, I would place a grid view of the 'Pieces in Locations' table filtered for that specific item. So it would show how many pieces of that item we have and where they are.

Is something similar already possible in Budibase? Or will it be in the future?

**Comment by aptkingston** [comment]
Hi @FloMiLe. With our latest release, you can now filter the results of a repeater, which would allow you to achieve what you're trying to do.

You could use a `Repeater` to list all the `Pieces in location` relationship, and then filter the repeater to only match a certain item you're looking for. If you've got the item ID in the URL, then you could filter using that.

Check out this video which shows how to filter a `Repeater`. This video is using a form input to filter, but you can filter with any values you want, including a URL parameter.

[https://www.youtube.com/watch?v=MLjFZ4VZ6Uk&feature=youtu.be](https://www.youtube.com/watch?v=MLjFZ4VZ6Uk&feature=youtu.be)
